Added dedicated java 8 build with latest log4j2 dependency. (#3076)
elastic/apm-agent-java: elastic-apm-agent-java8/README.md

# elastic-apm-agent-java8

> This project is a secondary build of `elastic-apm-agent` which ships the latest log4j2 version and therefore

Reviewer: ```suggestion
This project is a secondary build of `elastic-apm-agent` which ships a log4j2 version that```
